[onert] building onert failed

```
dragon@loki:~/Works/github/ONE$ BUILD_TYPE=Release TARGET_ARCH=armv7l CROSS_BUILD=1 make -f Makefile.template clean configure build install

...
Generating test(s) from spec: /home/dragon/Works/github/ONE/tests/nnapi/specs/V1_2/argmax_dynamic_nnfw.mod.py                                                                                                                                                                      
Traceback (most recent call last):                                                                                                                                                                                                                                                 
  File "/home/dragon/Works/github/ONE/tests/nnapi/nnapi_test_generator/android-10/cts_generator.py", line 298, in <module>                                    
    exec(open(tg.FileNames.specFile, "r").read())                                                                                                                   
  File "<string>", line 45, in <module>                                                                                                      
TypeError: __init__() missing 1 required positional argument: 'tensor_type'                                                                  
tests/nnapi/CMakeFiles/generate-runtime-nnapi-tests.dir/build.make:755: recipe for target '/home/dragon/Works/github/ONE/tests/nnapi/src/generated/all_generated_V1_2_cts_tests.cpp' failed
make[3]: *** [/home/dragon/Works/github/ONE/tests/nnapi/src/generated/all_generated_V1_2_cts_tests.cpp] Error 1                                                        
make[3]: Leaving directory '/home/dragon/Works/github/ONE/Product/armv7l-linux.release/obj'                                                                  
CMakeFiles/Makefile2:2910: recipe for target 'tests/nnapi/CMakeFiles/generate-runtime-nnapi-tests.dir/all' failed                            
make[2]: *** [tests/nnapi/CMakeFiles/generate-runtime-nnapi-tests.dir/all] Error 2                                                  
make[2]: *** Waiting for unfinished jobs....                                                                                                                  
```

```
dragon@loki:~/Works/github/ONE$ git log --oneline -1
9013898e (HEAD -> master, upstream/master, origin/master, origin/HEAD) Add tensor type argument for DynamicInputGenerator (#1807)
```

I'm not sure what I miss...

--- a/tests/nnapi/specs/V1_2/argmax_dynamic_nnfw.mod.py
+++ b/tests/nnapi/specs/V1_2/argmax_dynamic_nnfw.mod.py
@@ -42,7 +42,7 @@ model = Model()
 
 model_input_shape = [1, 2, 2, 2]
 
-dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input_shape)
+dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input_shape, "TENSOR_FLOAT32")
 
 test_node_input = dynamic_layer.getTestNodeInput()
 

--- a/tests/nnapi/specs/V1_2/div_dynamic_nnfw.mod.py
+++ b/tests/nnapi/specs/V1_2/div_dynamic_nnfw.mod.py
@@ -40,7 +40,7 @@ model = Model()
 
 model_input1_shape = [3, 4]   # first input shape of Div. 12 float32s
 
-dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input1_shape)
+dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input1_shape, "TENSOR_FLOAT32")
 
 test_node_input = dynamic_layer.getTestNodeInput() # first input of Div is dynamic tensor
 

--- a/tests/nnapi/specs/V1_2/mul_dynamic_nnfw.mod.py
+++ b/tests/nnapi/specs/V1_2/mul_dynamic_nnfw.mod.py
@@ -40,7 +40,7 @@ model = Model()
 
 model_input1_shape = [3, 4]   # first input shape of Mul. 12 float32s
 
-dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input1_shape)
+dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input1_shape, "TENSOR_FLOAT32")
 
 test_node_input = dynamic_layer.getTestNodeInput() # first input of Mul is dynamic tensor
 

